tests/system/lab/README.rst

```diff
@@ -0,0 +1,25 @@
+
+Script to automate FogLAMP Lab
+------------------------------
+
+1. Install git i.e. `sudo apt install git`
+
+2. Clone FogLAMP repo and `cd tests/system/lab/`
+
+3. Check and set the configuration in `test.config`
+
+For CI or individual's setup, `test.config` should be replaced (altered) per the parameters.
+
+Execute `./run` to run test once. Default version it will use is nightly, you can pass an argument e.g. `./run 1.7.0RC`
+To run the test for required (say 10) iterations or until it fails - execute `./run_until_fails 10 1.7.0RC`
+
+
+**`run` and `run_until_fails` use the following scripts in its execution:**
+
+**remove**: apt removes all foglamp packages; deletes /usr/local/foglamp; reboots
```

Comment:
I believe reboot is not required, as we are not changing any system hardware configuration in remove script